html/themes/custom/common_design_subtheme/components/rw-blog/rw-blog.css

```diff
@@ -0,0 +1,93 @@
+/**
+ * Blog post.
+ *
+ * Latest blog post.
+ *
+ * Id: #blog-post
```

Comment:
This ID has changed. It's now `blog` for the Section ID, and `blog_post` for children

Eg. `<article class="rw-river-article rw-river-article--blog_post">`